wcag2ict.js

```diff
@@ -102,6 +106,134 @@ function prepTerm(n) {
 	}
 }
 
+// number notes if there are multiple per section
+function numberNotes() {
+	var sectionsWithNotes = new Array();
+	document.querySelectorAll(".note").forEach(function(note) {
+		var container = note.closest("dd");
+		if (container == null) container = note.closest("blockquote");
+		if (container == null) container = note.closest("section");
+		sectionsWithNotes.push(container);
+	});
+	
+	sectionsWithNotes.forEach(function(sec) {
+		if (sec.noteprocessed) return;
+		var notes = sec.querySelectorAll('.note');
+		// no notes, shouldn't happen
+		if (notes.length == 0) return;
+		// one note
+		if (notes.length == 1) {
+			// respec note, do nothing
+			// included note, add marker
+			if (notes[0].querySelector(".marker span") == null) addNoteMarker(notes[0], "Note: ");
```

Comment:
Problem with algorithm. This is making Editor's notes numbered notes.

Reply:
NVM...the algorithm is good. I found a place where I had used "note" and note "ednote" for the formatting so it was my bug in the document.

Reply:
Hey @maryjom Thanks much for pointing this out. I've added some logic to discern which of these notes are not editor notes. I think this is better now.

Reply:
@daniel-montalvo Yes, that change works. I also added a change to remove the "Guidance When Applying" sections from the TOC for the glossary terms sections. This also significantly cleans up the TOC. I managed to copy existing similar code and figure out what needed to change to get this to work.